rstudio/rstudio
# Remove old ssh bits from Windows Desktop

### Intent

Addresses https://github.com/rstudio/rstudio-pro/issues/6880

### Approach

Stop installing the old msys-ssh bits and no longer put them on the path. Also, stop setting the `RSTUDIO_MSYS_SSH` environment variable (pointed to the msys-ssh bits).

Windows-10 and newer include OpenSSH (https://learn.microsoft.com/en-us/windows-server/administration/openssh/openssh-overview).

### Automated Tests

None

### QA Notes

Check that the following folders are not installed with RStudio on Windows:

`c:\program files\rstudio\resources\app\bin\msys-ssh-1000-18\`
`c:\program files\rstudio\resources\app\bin\msys_ssh\`

If you're really excited about it, test that you can successfully use subversion via ssh on Windows using OpenSSL in Windows (or the git implementation, etc.). I conducted my own test, setting up a subversion repo on my Mac and using it from Windows via ssh (with this fix in place), and confirmed I could use svn+ssh protocol.

### Documentation

Updated user guide as part of this PR.

### Checklist
